__tests__/api/chat/channels/read.test.js

```diff
@@ -45,13 +45,12 @@ describe('POST /chat/channels/read', () => {
     );
   });
 
-  test('should return 403 error validation if channelType is not chat or group', async () => {
+  test('should return 403 error validation if channel type is not provided', async () => {
     const response = await supertest(app)
       .post('/api/v1/chat/channels/read')
       .set({Authorization: `Bearer token`})
       .send({
-        message: 'Hello World',
-        channelType: 3
+        channelId: 'channel-id'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case is checking for a 403 error when the `channelType` is not provided. However, the `channelType` is not being sent in the request body. If the `channelType` is required, it should be included in the request body.
```diff
     .send({
+       channelType: 'chat',
        channelId: 'channel-id'
     })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->